launcher/ui/pages/global/LauncherPage.ui

```diff
@@ -303,13 +313,25 @@
             </property>
            </widget>
           </item>
-          <item row="1" column="0">
-           <widget class="QLabel" name="label_4">
+         </layout>
+        </widget>
+       </item>
+       <item>
+        <widget class="QGroupBox" name="toolsBox">
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
+         <property name="title">
+          <string>Tools</string>
+         </property>
+         <layout class="QVBoxLayout" name="verticalLayout">
+          <item>
+           <widget class="QCheckBox" name="preferMenuBarCheckBox">
             <property name="text">
-             <string>Colors</string>
-            </property>
-            <property name="buddy">
-             <cstring>themeComboBoxColors</cstring>
+             <string>Always show menu bar instead of tool bar (more keyboard friendly, less pretty)</string>
```

Comment:
I feel like this string is way too long.

```suggestion
             <string>Replace toolbar with menubar</string>
```

And add a tooltip saying
```
The menubar is more friendly for keyboard-driven interaction.
```
or something along those lines